### DISCUSSION TITLE
Accessibility with hotspots

### DISCUSSION THREAD

**Discussion by rebrob** [google/model-viewer#4807] [Q&A]
### Description

UNABLE TO TOGGLE ANNOTATIONS ON/OFF USING KEYBOARD: There are a lot of accessibility features built into model-viewer already, but there is some standard keyboard accessibility that is missing with regard to the hotspots. The user is able to tab to each hotspot, but not able to toggle the hotspot annotations ON/OFF with the keyboard (typically this would work with the Shift key or the Return key). 

Is this already on the radar for model-viewer devs? If so, is there a target delivery date?

Thanks for all the great work! 

Rebecca

#### Live Demo

This can be viewed in the Annotations demo directly on the model-viewer site. https://modelviewer.dev/examples/annotations/index.html

### Version

- https://unpkg.com/@google/model-viewer

### Browser Affected

- [ x] Chrome, version: xx.x.xxxx.xx
- [ x] Edge
- [x ] Firefox
- [x ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [x ] Linux
- [ ] MacOS
- [x ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
Since hotspots and related animations are defined in HTML/CSS by the user outside of MV, it's on you to add a11y features to them like any other feature on your website. I don't think there's a way for us to do it for you and ensure consistency with the rest of your a11y features.

**Reply by rebrob** [reply]
Fair enough. Thanks for your response.
